src/components/team/styledTeam.js

```diff
@@ -0,0 +1,59 @@
+import styled from "styled-components";
+import linkedin from '../../images/linkedin.svg';
+
+export const TeamWrapper = styled.section`
+    display: flex;
+    flex-direction: column;
+    width: 53.61%;  // 772/1440
+    margin: auto;
+    padding: 0 0 136px;
+`
+export const TeamBlocks = styled.ul`
+    display: flex;
+    justify-content: space-between;
+    margin: 80px auto 120.8px auto;
+    width: 100%;
+`
+export const TeamBlock = styled.li`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 25.91%;  // 200/772
+`
+export const TeamImage = styled.img`
+    width: 146px;
+    height: 146px;
+    margin-bottom: 20px;
+`
+export const TeamText = styled.p`
+    font-style: normal;
+    font-weight: normal;
+    font-size: 24px;
+    line-height: 28px;
+    text-align: center;
+`
+export const Link = styled.a`
+    width: 32px;
+    height: 32.2px;
+    background-image: url(${linkedin});
+    margin: 21px auto 0 auto;
+    cursor: pointer;
+`
+export const PartnerRow1 = styled.div`
+    width: 100%;
```

Comment:
the alignment of the items is off. Notice how Stewards and Ninja van are way higher than chef at work. I think align-items: center will do the trick. If it doesn't, try align-items plus adding a fixed height for the row.